libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -168,6 +177,11 @@ bool AP_Follow::get_target_dist_and_vel_ned(Vector3f &dist_ned, Vector3f &dist_w
         return false;
     }
 
+    // change to altitude above home if relative altitude is being used
+    if (target_loc.flags.relative_alt == 1){
```

Comment:
tiny little formatting issue, there should be a space before the curly bracket.  ie.. "== 1) {".